src/ui/reducers/app.ts

```diff
@@ -249,7 +249,7 @@ export const getIndexedProgress = createSelector(getLoadedRegions, regions => {
     return totalTime + end.time - begin.time;
   }, 0);
 
-  return totalIndexedTime / totalLoadingTime;
+  return totalIndexedTime ? 1 : 0;
```

Comment:
Can you add a comment

Reply:
Agreed. There's a comment above, that no longer seems relevant:
https://github.com/replayio/devtools/blob/70e3c115ed1a8f6e29bb945137afdb1f4d29e0ef/src/ui/reducers/app.ts#L225-L229

Reply:
nit: We could probably avoid some unnecessary iteration in this new approach, if we go with it.

```js
return indexed.find(({ begin, end }) => end.time - begin.time > 0) ? 1 : 0;
```